### PR TITLE
Add English version for #483

### PR DESCRIPTION
**Pull request by ikanago**
Ignore newlines automatically inserted by VSCode(or maybe Jupyter extension)... 🙏

### CODE REVIEW

File: doc/en/source/guide/2.0_python_advanced.ipynb [hunk 1]
```diff
@@ -3434,6 +3434,56 @@
     "gate = Measurement(target, classical_pos)\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### ProbabilisticInstrument\n",
+    "<code>Probabilistic</code>関数に加え、作用したゲートの添え字を取得することができる関数です。distributionの総和が1未満であるなどの理由でどのゲートも作用しなかった場合、gate_listの要素数がclassical_addressに保存されます。"
```
Comment (KowerKoint) [inline doc/en/source/guide/2.0_python_advanced.ipynb:3442]: translate

File: doc/en/source/intro/4.1_python_tutorial.ipynb [hunk 2]
```diff
@@ -553,7 +553,180 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qulacs.gate import TOFFOLI\n",
+    "control1 = 0\n",
+    "control2 = 1\n",
+    "target1 = 1\n",
```
Comment (KowerKoint) [inline doc/en/source/intro/4.1_python_tutorial.ipynb:580]:
```suggestion
    "target1 = 2\n",
```
This should be fixed also in Japanese version.

**Review by KowerKoint — CHANGES_REQUESTED** [review]

**Comment by ikanago** [comment]
Sorry for force pushing that leads vanishing your review...
I learned I should tidy commits by rebasing after review.

**Review by KowerKoint — APPROVED** [review]
Thank you.

**Pull request merged**